intranet/apps/eighth/views/admin/activities.py

```diff
@@ -148,6 +149,10 @@ def edit_activity_view(request, activity_id):
                     else:
                         messages.success(request, "You modified the default rooms, but those changes will not affect any scheduled activities.")
 
+                if set(old_room_ids) != set(new_room_ids):
+                    # Notify people that the activities they're signed up for have changed rooms
+                    room_changed_activity_email.delay(activity, activity.rooms.all(), EighthRoom.objects.filter(id__in=new_room_ids))
```

Comment:
```suggestion
                    room_changed_activity_email.delay(activity, old_rooms, EighthRoom.objects.filter(id__in=new_room_ids))
```

Since we already define it, we should use it.